### PR TITLE
Utility function to create time_point from UTC timestamp

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/common/utility/TimeService.cpp [hunk 1]
```diff
@@ -225,4 +225,8 @@ Opm::TimeStampUTC Opm::operator+(const Opm::TimeStampUTC& lhs, std::chrono::dura
     return Opm::TimeStampUTC( Opm::TimeService::advance(Opm::asTimeT(lhs) , delta.count()) );
 }
 
+Opm::time_point Opm::asTimePoint(const Opm::TimeStampUTC& ts)
+{
+    return Opm::TimeService::from_time_t( Opm::asTimeT(ts) );
```
Comment (bska) [inline src/opm/common/utility/TimeService.cpp:230]: If we're being pedantic, this function is already in the `Opm` namespace so should probably be implemented as
```C++
return TimeService::from_time_t(asTimeT(ts));
```
instead.

Comment (joakim-hove) [reply src/opm/common/utility/TimeService.cpp:230]: Well - don't ask my why; but the main bulk of implementation of the time service is outside the opm namespace - I am just following the pattern in the file. Happy to change it though.

Comment (bska) [reply src/opm/common/utility/TimeService.cpp:230]: > the main bulk of implementation of the time service is outside the opm namespace

I'm not sure I understand what you mean.  The `TimeService` namespace is nested within `Opm` and all `TimeService` functions are named
```
Opm::TimeService::something
```
so they're clearly within the `Opm` namespace.

**Comment by joakim-hove** [comment]
I've removed some `Opm::` I'll merge this now.

**Pull request merged**